src/features/app/components/Settings/Form.tsx

```diff
@@ -17,23 +17,25 @@ const SettingsForm = () => {
   const [isResponseSuccess, setIsResponseSuccess] = useState(false);
 
   const defaultValues = {
-    firstName: user.firstName,
-    lastName: user.lastName,
-    locale: user.locale,
+    firstName: user?.firstName,
+    lastName: user?.lastName,
+    locale: user?.locale,
   };
 
   const validationSchema = object().shape({
-    firstName: string().required(intl.messages['common.required']),
-    lastName: string().required(intl.messages['common.required']),
-    locale: string().nullable().required(intl.messages['common.required']),
+    firstName: string().required(intl.formatMessage({ id: 'common.required' })),
+    lastName: string().required(intl.formatMessage({ id: 'common.required' })),
+    locale: string()
+      .nullable()
+      .required(intl.formatMessage({ id: 'common.required' })),
   });
 
   const formMethods = useForm({
     defaultValues,
     resolver: yupResolver(validationSchema),
   });
 
-  const onSubmit = async (attributes) => {
+  const onSubmit = async (attributes: any) => {
```

Comment:
[Here's the doc](https://react-hook-form.com/ts/#SubmitHandler) from react-hook-form to type this submit handler, if you want to avoid use `any`